Дз5/css/style_for_form.css

```diff
@@ -0,0 +1,187 @@
+body {
+  background-color: #F4F0E8;
+  font-family: Comic Sans MS, cursive;
+  font-size: 12px;
+}
+
+h1 {
+  font-size: 17px;
+}
+
+p {
+  font-weight: bold;
+  color: #14A4A9;
+}
+
+/*------ in form fields---*/
+input::placeholder {
+  font-family: segoe print, cursive;
+  font-size: 10px;
+  color: #18E88D;
+}
+
+/* --- Оцените наш сайт--- */
+.grade>input {
+  position: absolute;
+  opacity: 0;
+}
+
+.grade>span::before {
+  content: '';
+  display: inline-block;
+  width: 10px;
+  height: 10px;
+  border: 1px solid #B1B4B7;
+  border-radius: 50%;
+  margin-right: 5px;
+  background-color: #FFFFFF;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: 70% 70%;
+}
+
+.grade>input:checked+span::before {
+  border-color: #FDFDFE;
+  border-width: 1px;
+  background-color: #0B76EF;
+  background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3e%3ccircle r='3' fill='%23fff'/%3e%3c/svg%3e");
+}
+
+/*------ Как вы проводите свое свободное время -----*/
+.leisure>input {
+  position: absolute;
+  opacity: 0;
+}
+
+.leisure>span::before {
+  content: '';
+  display: inline-block;
+  width: 10px;
+  height: 10px;
+  border: 1px solid #adb5bd;
+  border-radius: 3px;
+  margin-right: 5px;
+  background-color: #FFFFFF;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: 70% 70%;
+}
+
+.leisure>input:checked+span::before {
+  border-color: #0b76ef;
+  background-color: #0b76ef;
+  background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 8 8'%3e%3cpath fill='%23fff' d='M6.564.75l-3.59 3.612-1.538-1.55L0 4.26 2.974 7.25 8 2.193z'/%3e%3c/svg%3e");
+}
+
+/*-----Какие стили музыки вы слушаете -----*/
+option {
+  font-family: Comic Sans MS, cursive;
+  ;
+  font-size: 12px;
+}
+
+select {
+  width: 200px;
+}
+
+select::-webkit-scrollbar {
+  width: 10px;
+}
+
+select::-webkit-scrollbar-thumb {
+  border-radius: 5px;
+  background-color: gray;
+}
+
+/*----Ползунок ---*/
+input[type="range"] {
+  -webkit-appearance: none;
+  background: transparent;
+  cursor: pointer;
+  width: 200px;
+}
+
+input[type="range"]::-webkit-slider-runnable-track {
+  background-color: #C3D2CB;
+  border-radius: 20px;
+  height: 8px;
+}
+
+input[type="range"]::-webkit-slider-thumb {
+  appearance: none;
+  margin-top: -9px;
+  background-color: #18E88D;
+  height: 25px;
+  width: 25px;
+  border-radius: 50%;
+}
+
+/*------Включить-----*/
+.switch {
+  position: relative;
+  width: 200px;
+  height: 50px;
+}
+
+.wrapper,
+.position1,
+.position2 {
```

Comment:
we can style it just with lable, without any div's
https://prnt.sc/tgn_UKPh3bdg
just use lable and depending on input, on checked or unchecked just change background and position before element